### PR TITLE
Initial commit for the Q4 4.13 upgrade

### PR DESCRIPTION
**Pull request by tbaker1313**
Not for posting until September 25th

### CODE REVIEW

**Review by StevenBarre — APPROVED** [review]
Looks good to me, don't post until Monday though. Sal still needs to talk to the change freeze people.

File: notifications/09-20-23-1500.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+CHG0052480 - MCS GOLDDR - 2023 Q4 Patching - Upgrade to OCP 4.13.12
+
+These six changes represent the work involved to upgrade the production clusters from Openshift 4.12 to 4.13.
+
+**When?**
+
+CHG0052476 - MCS SILVER - 2023 Q4 Patching - Hard Prune of Image registry, ETCD Defrag and Trident Upgrade will commence at 0600 on Sunday October 15th and run as late as 0900.
+
+CHG0052425 - MCS SILVER - 2023 Q4 Patching - Upgrade to OCP 4.13.12 will commence at 9:00am on Monday October 16th. Upgrade efforts will be allowed to proceed between 9:00am and 5:00pm, and be paused after-hours to ensure developers are present during node drains to look after their apps if needed.
```
Comment (wmhutchison) [inline notifications/09-20-23-1500.md:21]: I recommend adding when the change window ends for each RFC involving regular Openshift upgrades. It's more or less done for the Sunday morning RFCs.

**Review by wmhutchison — APPROVED** [review]
👍

File: notifications/09-20-23-1500.md (hunk 1 above)
Comment (tbaker1313) [reply notifications/09-20-23-1500.md:21]: I'll add the end dates in a moment.

**Review by wmhutchison — APPROVED** [review]
LGTM

**Pull request merged**